### ISSUE TITLE
Mark fields with errors.

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#481]
# Fields

Visibility: 3
Evaluation: Partially Supports
Success Criteria: 3.3.1: Error Identification
Needs design: 

### Description
Mark fields that fail validation with aria-invalid.

https://www.w3.org/TR/wai-aria-1.1/#aria-invalid

![]()

**Comment by amazingphilippe** [comment]
This may have had an impact for this finding if the invalid attribute wasn't set:

https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwBYyaQxOttZ0IAw/rec3AJoQglDnjvh7a?blocks=hide

> The error message for the 'Sending email address' field didn't describe what the user needed to do to correct their mistake. For a participant using a screen reader, the error message didn't receive focus right away and so they didn't know there was an error.
